homeassistant/components/flux_led/__init__.py

```diff
@@ -49,17 +47,28 @@ def async_wifi_bulb_for_host(host: str) -> AIOWifiLedBulb:
     return AIOWifiLedBulb(host)
 
 
+@callback
+def async_name_from_discovery(device: dict[str, Any]) -> str:
+    """Convert a flux_led discovery to a human readable name."""
+    if mac := device.get(ATTR_ID):
```

Comment:
You could turn all these if-statements around and get slightly cleaner code. 

```
if mac is None:
   return ip
```

etc

Reply:
d60679d76410d0df7881246310fa8110f6be3881